stdlib/REPL/src/TerminalMenus/RadioMenu.jl

```diff
@@ -93,6 +96,10 @@ function keypress(m::RadioMenu, i::UInt32)
     return true
 end
 
+accepts_j(m::RadioMenu) = !('j' in m.keybindings || 'k' in m.keybindings)
```

Comment:
This might be too expensive. We can perform this check once inside the constructor and then save a Boolean flag somewhere. I would rather store this flag in the menu (not the configuration) since it isn't quite user-configurable but a trying-to-be-smart default. @timholy what do you think about this?

Reply:
Sounds reasonable. Though I doubt this is performance sensitive: at 120wpm, the gap between characters is ~0.1s. It would have to be a long list for you to notice that!